### PR TITLE
Ful logs documentation updates

### PR DESCRIPTION
**Pull request by fulyauluturk**
Updated Logs --> Overview and Generic Logs integration pages.

### CODE REVIEW

**Review by otisg — CHANGES_REQUESTED** [review]
Suggested some changes

File: docs/logs/sending-log-events.md [hunk 1]
```diff
@@ -62,3 +62,270 @@ Alternatively, you can explore various custom integration options for shipping l
...
+<div>For iOS apps use <a href="https://github.com/sematext/sematext-logsene-ios" target="_blank" rel="noopener">Sematext Logs for iOS</a> library.</div>
+
+### Android 
+
+<div>For Android apps use<a href="https://github.com/sematext/sematext-logsene-android" target="_blank" rel="noopener"> Sematext Logs for Android</a> library.</div>
+
+### AWS EC2 
+
+If you're an EC2 user, you can log Sematext from your instances by
+setting up a <a class="" href="https://apps.sematext.com/ui/logs#">log
```
[the remark this answers is not in the document]
Comment (fulyauluturk) [reply docs/logs/sending-log-events.md:138]: linking where? didn't understand

**Comment by fulyauluturk** [comment]
FYI: Created https://sematext.atlassian.net/browse/SC-18464 . So once we fix it the link to howto pages should work properly.

File: mkdocs.yml [hunk 2]
```diff
@@ -193,6 +193,9 @@ pages:
         - Generic Logs: integration/generic-logs-integration.md
         - Apache Logs: integration/apache-logs.md
         - AWS ECS: integration/ecs-logs.md
```
Comment (otisg) [inline mkdocs.yml:195]: maybe you want aws- prefix, but if PITA to rename now just ignore me.

File: docs/logs/sending-log-events.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (otisg) [reply docs/logs/sending-log-events.md]: linking to https://apps.sematext.com/ui/logs# from the docs page

**Review by otisg — APPROVED** [review]

File: mkdocs.yml (hunk 2 above)
Comment (fulyauluturk) [reply mkdocs.yml]: done

**Pull request merged**